src/compas/geometry/objects/line.py

```diff
@@ -144,6 +144,17 @@ def __iter__(self):
     def __eq__(self, other):
         raise NotImplementedError
 
+    # ==========================================================================
+    # queries
+    # ==========================================================================
+
+    def point(self, t):
+        """Point: The point from the start to the end at a specific normalized parameter."""
+        if t < 0 or t > 1:
+            return None
+        v = self.direction * (t * self.length)
```

Comment:
should we not also check for the bounds and then return one of the points directly if appropriate?

Reply:
ok